src/components/landing/rework/footer-new.tsx

```diff
@@ -15,39 +17,45 @@ export function FooterNew() {
       <Separator />
       <FooterNavNew />
       <Separator />
-      <div>{renderFooterLine()}</div>
+      {renderFooterLine()}
     </footer>
   )
 
   function renderFooterLine() {
+    const items = R.concat(
+      R.map(
+        R.assoc('mdHidden', false),
+        footerData.footerNavigation[2].children
+      ),
+      R.map(R.assoc('mdHidden', true), footerData.footerNavigation[1].children)
+    )
```

Comment:
Hmm to be honest this is quit eunreadable for me since I never really used ramda.
Atm. we only use ramda twice in the frontend. 
We should make a conscious decision if we want to rely on it more. 

My personal preference would be to use it only when it sparingly, since it's another library new people might have to learn when they want to contribute.

Reply:
(not that the previous iteration was very readable ^^)

Reply:
As soon as you understand that ramda supports currying and partial application, this code actually makes sense.

(side note below)

The basic idea is good, why not define the function as

```jsx
function renderFooterLineItems(
  items: {title: string; url: string}[]
  mdHidden: boolean 
) {
 return items.map(...)
}
```
and call it like

```jsx
<>
  {renderFooterLineItems(footerData.footerNavigation[2].children, false)}
  {renderFooterLineItems(footerData.footerNavigation[1].children, true)}
</>
```